### DISCUSSION TITLE
how to know why load error

### DISCUSSION THREAD

**Discussion by carl1990** [coil-kt/coil#2069] [Q&A] [answered]
`    AsyncImage(
        model = "http://pic-bucket.ws.126.net/photo/0003/2021-11-16/GOTKEOOU00AJ0003NOS.jpg",
        placeholder = painterResource(R.drawable.icon_printer),
        error = painterResource(R.drawable.icon_notification),
        modifier = Modifier
            .fillMaxHeight()
            .fillMaxWidth()
            .padding(start = 36.dp, end = 36.dp),
    )`

will show error icon,How can i know reason?

**Comment by asyl-dev** [comment, marked as answer]
``` kotlin
	AsyncImage(
		model = imageRequest
			.build(),
		onLoading = { loading ->
			//do something
		},
		onSuccess = { success ->
			//do something
		},
		onError = { error ->
			//do something
		},
		contentDescription = contentDescription
	)
```
